Providers/StaminaProviderConfig.jsx

```diff
@@ -0,0 +1,119 @@
+// /* eslint-disable react/destructuring-assignment */
+import React, { Component, createContext } from "react";
+import { AsyncStorage } from "react-native";
+import axios from "react-native-axios";
+
+const { Provider, Consumer } = createContext();
+
+class StaminaProvider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      stamina: null,
+      date: null
+    };
+  }
+
+  async componentDidMount() {
+    const storedResult = await this.loadFromStorage();
+    try {
+      const resp = await axios.get("http://worldtimeapi.org/api/ip");
```

Comment:
this logic should be abstracted to a separate function, so you don't have to hardcode the url in 4 places. Maybe put it in `expo constants`